BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.h

```diff
@@ -91,6 +91,7 @@ extern NSString *const BOXOAuth2AuthDelegationNewClientKey;
 // Notifications
 extern NSString *const BOXUserWasLoggedOutDueToErrorNotification;
 extern NSString *const BOXAuthOperationDidCompleteNotification;
+extern NSString *const BOXAccessTokenRefreshDiagnosisNotification;
```

Comment:
Are we treating this as part of our public interface? If not, should it be commented as private?

Reply:
done